importexport/musicxml/importxmlfirstpass.cpp

```diff
@@ -92,6 +92,19 @@ void MusicXmlPart::calcOctaveShifts()
             }
       }
 
+bool MusicXmlPart::isVocalStaff() const
+      {
+      QList<QString> vocalInstrumentNames({"Voice",
```

Comment:
I'd suggest to use an std::vector here and also make it static, because you don't really need to allocate it on every call

it also makes sense to put it at the top of .cpp as a static constant variable

Reply:
Good call!